indispensable-asn1/build.gradle.kts

```diff
@@ -1,114 +1,43 @@
 import at.asitplus.gradle.*
-import org.jetbrains.kotlin.gradle.ExperimentalWasmDsl
 
 
 plugins {
-    id("com.android.library")
-    kotlin("multiplatform")
-    kotlin("plugin.serialization")
-    id("signing")
-    id("at.asitplus.gradle.conventions")
+    id("at.asitplus.signum.buildlogic")
 }
 
-val artifactVersion: String by extra
-version = artifactVersion
-
+signumConventions {
+    android("at.asitplus.signum.indispensable.asn1")
+    mavenPublish(
+        name = "Indispensable ASN.1",
+        description = "Kotlin Multiplatform ASN.1 Engine"
+    )
+}
```

Comment:
So since this does not set `supreme = false`, does that mean the setter never runs and `project.version` is never set?
(This is a comment on the sagacity of putting that logic in a setter.)